sdk/src/types/globals.ts

```diff
@@ -1,5 +1,5 @@
 // using "| any" to ensure compatibility with various "Fetch" alternative function signatures
-export type FetchInterface = (input: string | any, init?: RequestInit | any) => Promise<unknown>;
+export type FetchInterface = (input: string | any, init?: RequestInit | any) => Promise<Response>;
```

Comment:
Unfortunately this is not something we can do, it must stay as `unknown` to ensure non-native fetch implementations (e.g. `ofetch`) are supported
```suggestion
export type FetchInterface = (input: string | any, init?: RequestInit | any) => Promise<unknown>;
```

Reply:
Just a question: should I let simply `Promise<unknown>` or be more specific with  `Promise<Response | unknown>`

Reply:
I see that has already been done, resolving.